portfolio/src/main/java/com/google/sps/servlets/BasketballCourtsServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.BasketballCourtsDatastore;
+import com.google.sps.data.BasketballCourt;
+
+/** Returns the accessible basketball courts in New York City */
+@WebServlet("/basketball-courts")
+public final class BasketballCourtsServlet extends HttpServlet {
+    
+  BasketballCourtsDatastore basketballCourtsDatastore;
+  List<BasketballCourt> accessibleCourts;
```

Comment:
you should not cache it here. let the datastore handle it and call getAccessibleCourts each time.